README.md

```diff
@@ -1 +1,128 @@
-# grid3_client_go
\ No newline at end of file
+# **grid3_go:**
+
+grid3_go is a go client created to interact with threefold grid. It should manage CRUD operations for deployments on the grid.
+
+- ## **Grid3_go flow:**
+
+  1. the deployment manager will be initialized with identity/network information.
+  2. deployment manager will expose the `deploy` which the user will call it and give it old/new deployments
```

Comment:
exported functions start with an uppercase `Deploy`

Reply:
also it's very weird that Deploy just gives the old/new deployments, which i honestly don't understand at all. Is it the existing deployments of the user on a node or on all nodes?  where is its state stored? why do we need to do that?

Reply:
It's all handled in deploy function which is called in Deploy. 
It loads old deployments from the grid, having their node id and contract id.
check new deployments , then handles the creation of new deployments, updating existing ones and deleting others.